eventuate-messaging-kafka-basic-consumer/src/main/java/io/eventuate/messaging/kafka/basic/consumer/KafkaConsumerOffsetStorageStrategy.java

```diff
@@ -0,0 +1,13 @@
+package io.eventuate.messaging.kafka.basic.consumer;
+
+import java.util.Map;
+import org.apache.kafka.clients.consumer.OffsetAndMetadata;
+import org.apache.kafka.common.TopicPartition;
+
+public interface KafkaConsumerOffsetStorageStrategy {
```

Comment:
This functionality is not used in this project and so should be moved somewhere else